website/content/docs/concepts/seal.mdx

```diff
@@ -248,7 +248,9 @@ any storage backend.
    Wait for a little while for the migration information to replicate to all the
    nodes in case of Integrated Storage. In enterprise Vault, switching an Auto seal
    implies that the seal wrapped storage entries get re-wrapped. Monitor the log
-   and wait until this process is complete (look for `seal re-wrap completed`).
+   and wait until this process is complete (look for `seal re-wrap completed`). 
+
+   Note about seal-rewrap: Seal-rewrap gets invoked when changes in seal stanza is made, this is true even when you are trying to migrate from the same auto-unseal type like pkcs11 to pkcs11.
```

Comment:
```suggestion
<Warning heading="Seal configuration changes will invoke rewrap">

   Any change to the `seal` stanza in your Vault configuration invokes seal-rewrap,
   even "migrations" from the same auto-unseal type like `pkcs11` to `pkcs11`.

</Warning>
```

Style correction: write in active voice, make important information easy to see at a glance